Discuss other variants Fedora immutable desktops (#2350)
privacyguides/privacyguides.org: docs/desktop.md

    ![Fedora Silverblue logo](assets/img/linux-desktop/fedora-silverblue.svg){ align=right }

>     **Fedora Silverblue** is an immutable variants of Fedora with a strong focus on container workflows. Silverblue comes with the [GNOME](https://www.gnome.org/) desktop environment while [Kinoite](https://fedoraproject.org/kinoite/) comes with [KDE](https://kde.org/). There are other variants too including [Sericea](https://fedoraproject.org/sericea/) which is distributed with the [Wayland](https://wayland.freedesktop.org) only tiling window manager, [Sway](https://swaywm.org/). We don't recommend [Onyx](https://fedoraproject.org/onyx/) at this time as it still [requires X11](https://buddiesofbudgie.org/blog/wayland). All of these variants follow the same release schedule as Fedora Workstation, benefiting from the same fast updates and staying very close to upstream.

Reviewer: "is an immutable variants" needs to be changed of course, but I'd probably reword this paragraph entirely like:

```suggestion
    **Fedora Silverblue** is an immutable variant of Fedora with a strong focus on container workflows and the [GNOME](https://www.gnome.org/) desktop environment. If you prefer an environment other than GNOME, there are also other variants including [Kinoite](https://fedoraproject.org/kinoite/) (which comes with [KDE](https://kde.org/)) and [Sericea](https://fedoraproject.org/sericea/) (which comes with [Sway](https://swaywm.org/), a [Wayland](https://wayland.freedesktop.org)-only tiling window manager). We don't recommend [Onyx](https://fedoraproject.org/onyx/) at this time as it still [requires X11](https://buddiesofbudgie.org/blog/wayland). All of these variants follow the same release schedule as Fedora Workstation, benefiting from the same fast updates and staying very close to upstream.
```
